commit 90d81afaed5757587234f8edf2f501a8afccae3c
Author: Russell Senior <russell@personaltelco.net>
Date:   Sun Jul 28 01:54:26 2019 -0700

    gpsd: update to v3.19
    
    From gpsd NEWS:
    
    3.19: 2019-07-01
      Rearrange rawdata_t.  Bump the API because gps_data_t changed.
      Report sequence-ID fields in type 7 and 13 AIS messages.
      Preliminary support for SiRFstar V
      Improve error modeling.
      Update pseudo NMEA to v3.0, with fractional time.
      Improve cycle detection, mostly for NMEA.
      Move epe from gps_data_t to be near its friends in gps_fix_t.
      Preliminary u-blox 9 support.
      Add qErr in gps_data_t to store PPS quantization error.
      Add Android (AOSP) support
      Improved multi gnss and multi signal support.
      NMEA 4.10 multi gnss multi signal support.
      The arguments to "ubxtool -p P" have been expanded and changed.
      New options, -g, -x, -z, added to ubxtool for u-blox 9 support.
    
    3.18.1: 2018-10-19 (Gary E. Miller <gem@rellim.com>)
      Fix some installation issues.
      A few minor updates to ubxtool and driver_ubx.
      Add contrib/skyview2svg
    
    3.18: 2018-10-02 (Gary E. Miller <gem@rellim.com>)
      Add ECEF support to ievermore, italk,Skytraq, SiRF, U-blox drivers.
      Add ECEF support to JSON, cgps and xgps.
      Add GREIS (Javad) driver from Virgin Orbit.
      Add CLI tools zerk and ubxtool to manage JAVAD and u-blox GPS.
      Add gnssid:svid to satellite_t, cgps and xgps.  PRN will die.
      Add gnssid:svid to JSON, cgps and xgps.
      Add stricter version checking (more to todo).
      More and better regression tests.
      Better Python dependency checking, at build time and runtime.
      Fix several buffer issues.
      New polar plots, and improved statistice, in gpsprof.
      gpsd master/slave mode works, first time ever.
      All isnan() changed to !isfinite(), fixing many bugs.
      Client-side Python libraries may automatically reconnect
      Too many other bug fixes and improvements to mention.
      Over 1,000 commits from 46 different commiters.
    
    Signed-off-by: Russell Senior <russell@personaltelco.net>

diff --git a/utils/gpsd/Makefile b/utils/gpsd/Makefile
index dd9d9a9d5..55c975d0a 100644
--- a/utils/gpsd/Makefile
+++ b/utils/gpsd/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=gpsd
-PKG_VERSION:=3.17
-PKG_RELEASE:=3
+PKG_VERSION:=3.19
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SAVANNAH/$(PKG_NAME)
-PKG_HASH:=68e0dbecfb5831997f8b3d6ba48aed812eb465d8c0089420ab68f9ce4d85e77a
+PKG_HASH:=27dd24d45b2ac69baab7933da2bf6ae5fb0be90130f67e753c110a3477155f39
 
 PKG_MAINTAINER:=Pushpal Sidhu <psidhu.devel@gmail.com>
 PKG_LICENSE:=BSD-3-Clause
@@ -24,7 +24,7 @@ include $(INCLUDE_DIR)/scons.mk
 
 define Package/gpsd/Default
   DEPENDS+= +libusb-1.0
-  URL:=http://catb.org/gpsd/
+  URL:=https://gpsd.gitlab.io/gpsd/
 endef
 
 define Package/gpsd/Default/description
diff --git a/utils/gpsd/patches/010-musl.patch b/utils/gpsd/patches/010-musl.patch
deleted file mode 100644
index a2ba95112..000000000
--- a/utils/gpsd/patches/010-musl.patch
+++ /dev/null
@@ -1,40 +0,0 @@
---- a/driver_ais.c
-+++ b/driver_ais.c
-@@ -24,6 +24,7 @@
- 
- /* strlcpy() needs _DARWIN_C_SOURCE */
- #define _DARWIN_C_SOURCE
-+#define _BSD_SOURCE
- 
- #include <stdlib.h>
- #include <string.h>
---- a/gpsctl.c
-+++ b/gpsctl.c
-@@ -15,6 +15,7 @@
- 
- /* strlcpy() needs _DARWIN_C_SOURCE */
- #define _DARWIN_C_SOURCE
-+#define _BSD_SOURCE
- 
- #include <stdio.h>
- #include <stdlib.h>
---- a/gpsd_json.c
-+++ b/gpsd_json.c
-@@ -24,6 +24,7 @@ PERMISSIONS
- #define __DARWIN_C_LEVEL 200112L
- /* strlcpy() needs _DARWIN_C_SOURCE */
- #define _DARWIN_C_SOURCE
-+#define _BSD_SOURCE
- 
- #include <stdio.h>
- #include <math.h>
---- a/libgpsd_core.c
-+++ b/libgpsd_core.c
-@@ -20,6 +20,7 @@
- 
- /* strlcpy() needs _DARWIN_C_SOURCE */
- #define _DARWIN_C_SOURCE
-+#define _BSD_SOURCE
- 
- #include <time.h>
- #include <stdio.h>
